fog/sql_recovery_db/src/lib.rs

```diff
@@ -935,6 +935,78 @@ impl SqlRecoveryDb {
         }
     }
 
+    /// Get ETxOutRecords for a given ingress key from a block, and subsequent
+    /// blocks, up to some limit. (This is like a batch call to
+    /// get_tx_outs_by_block_and_key_retriable with lookahead, and makes
+    /// sense if there is high network latency.)
+    ///
+    /// Arguments:
+    /// * ingress_key: The ingress key we need ETxOutRecords from
+    /// * block_index: The first block we need ETxOutRecords from
+    /// * block_count: How many subsequent blocks to also request data for.
```

Comment:
Maybe this should be how many consecutive blocks to also request data for? Subsequent seems to suggest 0 should just return only this block_index, and 1 means return this and 1 additional?